libnetdata/socket/socket.c

```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 
 #include "../libnetdata.h"
+#include "../../web/server/web_client.h"
```

Comment:
Why library (files in `libnetdata` are libraries, aren't they?) is referencing something that is not a library? For me this is bad practice and shouldn't happen.

Reply:
Removed, no longer needed.